fe/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1257,4 +1271,8 @@ public void logReplaceTempPartition(ReplacePartitionOperationLog info) {
     public void logSetReplicaStatus(SetReplicaStatusOperationLog log) {
         logEdit(OperationType.OP_SET_REPLICA_STATUS, log);
     }
+
+    public void logRemoveExpiredAlterJob(RemoveAlterJobOperationLog log) {
```

Comment:
public void logRemoveExpireAlterJobV2(RemoveAlterJobV2OperationLog log) to avoid mislead?

Reply:
OK